packages/v4/patternfly-docs/patternfly-docs.source.js

```diff
@@ -76,6 +80,9 @@ module.exports = (sourceMD, sourceProps) => {
   sourceMD(path.join(logViewerContentBase, '/**/examples/*.md'), 'react');
   sourceMD(path.join(logViewerContentBase, '/**/demos/*.md'), 'react-demos');
 
+  // React-topology MD
+  sourceMD(path.join(reactTopologyPath, '/**/components/**/examples/*.md'), 'react');
```

Comment:
```suggestion
  sourceMD(path.join(reactTopologyPath, '/**/examples/*.md'), 'react');
```
I'm thinking we'll end up wanting docs for things outside of components as well